samples/04_gis_analysts_data_scientists/chennai_floods_analysis.ipynb

```diff
@@ -790,7 +790,7 @@
    "source": [
```

Comment:
<p>The human impact on the Pallikaranai marshlands is all too apparent in satellite imagery, as the marshland has shrunk to less than a third of its size in just two decades. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>